Source/Views/StackViewController.swift

```diff
@@ -24,7 +30,7 @@ final class StackViewController: UIViewController {
         self.preserveParentWidth = configuration.preserveParentWidth
         
         super.init(nibName: nil, bundle: nil)
-
+        
```

Comment:
I guess we removed trailing space from swiftlint and @AndreasThenn has a different Xcode setting